modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+
+Argo CD application::
+An application that tracks the continuous deployment of individual Kubernetes resources from the {gitops-shortname} repository, where the resources are defined as manifests, to a target Kubernetes cluster.
+
+`ArgoCD` CRD::
+A Kubernetes CRD that describes the wanted state for a given Argo CD cluster that allows you to configure the components that make up an Argo CD cluster.
+
+Argo CD instance::
+A single installation of Argo CD within a namespace, encapsulating all of the stateful aspects of a running Argo CD.
+
+Argo CD project::
+An entity within Argo CD that refers to the Argo CD open source project's link:https://argo-cd.readthedocs.io/en/stable/user-guide/projects[specific concept of projects], and the corresponding link:https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#projects[`AppProject` CR].
++
+You might deploy Argo CD applications in an Argo CD project across different namespaces or clusters. In contrast, an link:https://docs.openshift.com/container-platform/latest/getting_started/openshift-overview.html#getting-started-openshift-common-terms_openshift-overview[OpenShift project] is restricted to a single namespace and is equivalent in concept to a namespace.
++
+Argo CD project controls the behavior of Argo CD by restricting access to Git repository and remote clusters. Examples include using Argo CD project to control users by restricting who can access certain Argo CD applications or cluster resources through Argo CD UI or Argo CD CLI.
+
+Argo CD repository server (Argo CD-repo-server)::
+An Argo CD component that reads from Git repositories, generates corresponding application manifests, runs custom configuration management tooling, and returns the result to the Argo CD Application Controller.
+
+Argo CD resource (`ArgoCD` CR)::
+A CR that describes the wanted state for a given Argo CD instance that allows you to configure the components and settings which make up that Argo CD instance.
+
+Argo CD server (Argo CD-server)::
+A server that provides the API and UI for Argo CD.
+
+Argo Rollouts::
+A controller that you can use for managing the progressive deployment of applications hosted on Kubernetes and {OCP} clusters. This controller has a set of CRDs that provides advanced deployment capabilities such as blue-green, canary, canary analysis, and experimentation.
+
+Cluster-scoped instance::
+A mode in which Argo CD is configured to manage all resources on the cluster including certain cluster-specific resources such as cluster configuration, cluster RBAC, Operator resources, platform Operators, or secrets.
+
+Control plane ({gitops-shortname} control plane)::
+In the {gitops-shortname} context, you can have a control plane for every Argo CD you install. A {gitops-shortname} control plane is any namespace where you can install an Argo CD. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. 
```

Comment:
```suggestion
In the {gitops-shortname} context, you can have a control plane for every Argo CD instance you install. A {gitops-shortname} control plane is any namespace where you can install an Argo CD instance. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. 
```

Reply:
Suggestion changes the technical meaning. I am retaining content as is.